docs/user/TerratestDockerUsage.md

```diff
@@ -0,0 +1,98 @@
+# Using the Terratest Docker Container
+
+Use the Terratest Docker container to run the suite of Terratest Go tests. For more information on Terratest, follow the [Documentation](https://terratest.gruntwork.io/docs/) page. The Terratest Docker image is used by the [Github Workflow](../../.github/workflows/default_plan_unit_tests.yml) as a required check before merging changes.
+
+## Prereqs
+
+- Docker [installed on your workstation](../../README.md#docker).
+
+## Preparation
+
+### Docker image
+
+Run the following command to create the `viya4-iac-azure-terratest` Docker image using the provided [Dockerfile.terratest](../../Dockerfile.terratest)
+
+```bash
+docker build -t viya4-iac-azure-terratest -f Dockerfile.terratest .
+```
+
+The Docker image `viya4-iac-azure-terratest` will contain Terraform and Go executables, as well as the required Go modules. The Docker entrypoint for the image is `go test` and accepts several optional command line arguments. For more information on command line arguments, please see [Command Line Arguments](#command-line-arguments).
+
+### Docker Environment File for Azure Authentication
+
+Follow either one of the authentication methods described in [Authenticating Terraform to access Azure](./TerraformAzureAuthentication.md) and create a file with the authentication variable values to use with container invocation. Store these values outside of this repo in a secure file, for example
+`$HOME/.azure_docker_creds.env.` Protect that file with Azure credentials so only you have read access to it. **NOTE:** Do not use quotes around the values in the file, and make sure to avoid any trailing blanks!
+
+Now each time you invoke the container, specify the file with the [`--env-file`](https://docs.docker.com/engine/reference/commandline/run/#set-environment-variables--e---env---env-file) option to pass on Azure credentials to the container.
+
+### Docker Volume Mounts
+
+Add volume mounts to the `docker run` command for all files and directories that must be accessible from inside the container.
+- `--volume="$(pwd)":/viya4-iac-azure` the project must be mounted to the /viya4-iac-azure directory.
+
```

Comment:
Run the following command:
`--volume="$(pwd)":/viya4-iac-azure` 
Note that the project must be mounted to the `/viya4-iac-azure` directory.